-------------------- Log Start | Sun Apr 20 10:46:59 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds
config_file = /home/klippy/printer_data/config/printer.cfg

[authorization]
trusted_clients = 
	192.168.178.0/24
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
Loading Sqlite database provider. Sqlite Version: 3.40.1
Unsafe Shutdown Count: 6

System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
Module libcamera unavailable, import failed
Klipper Version: 0ba9ab7
2025-04-20 10:46:59,660 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 10:46:59,661 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.85ms
2025-04-20 10:46:59,661 [websockets.py:open()] - Websocket Opened: ID: 132642827222272, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 10:46:59,750 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 10:47:09,917 [websockets.py:on_close()] - Websocket Closed: ID: 132642827222272 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.25
2025-04-20 10:47:59,064 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 10:47:59,064 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.89ms
2025-04-20 10:47:59,064 [websockets.py:open()] - Websocket Opened: ID: 132642827217280, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 10:47:59,066 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 10:48:00,169 [websockets.py:on_close()] - Websocket Closed: ID: 132642827217280 Close Code: None, Close Reason: None, Pong Time Elapsed: 1.10
2025-04-20 10:48:00,171 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 10:48:00,172 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.71ms
2025-04-20 10:48:00,172 [websockets.py:open()] - Websocket Opened: ID: 132642827221696, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 10:48:00,177 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 10:48:00,498 [websockets.py:on_close()] - Websocket Closed: ID: 132642827221696 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.33
2025-04-20 10:48:00,501 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 10:48:00,502 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.74ms
2025-04-20 10:48:00,502 [websockets.py:open()] - Websocket Opened: ID: 132642827222080, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 10:48:00,505 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 10:48:10,916 [websockets.py:on_close()] - Websocket Closed: ID: 132642827222080 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.41
2025-04-20 10:49:38,839 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.178.26
2025-04-20 10:49:39,038 [application.py:log_request()] - 404 GET /favicon.ico (192.168.178.26) [_TRUSTED_USER_] 2.28ms
2025-04-20 10:55:04,401 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 10:55:04,503 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 10:55:04,521 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 10:55:06 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 10:55:06,390 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 183 B
2025-04-20 10:55:06,390 [server.py:launch_server()] - Server Config Error
Traceback (most recent call last):
  File "/home/klippy/moonraker/moonraker/server.py", line 562, in launch_server
    server = Server(app_args, log_manager, eventloop)
  File "/home/klippy/moonraker/moonraker/server.py", line 86, in __init__
    self.config = config = self._parse_config()
  File "/home/klippy/moonraker/moonraker/server.py", line 164, in _parse_config
    config = confighelper.get_configuration(self, self.app_args)
  File "/home/klippy/moonraker/moonraker/confighelper.py", line 1118, in get_configuration
    raise ConfigError("No section [server] in config")
moonraker.confighelper.ConfigError: No section [server] in config
2025-04-20 10:55:06,390 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 10:55:07 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 10:55:07,508 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 183 B
2025-04-20 10:55:07,508 [server.py:launch_server()] - Server Config Error
Traceback (most recent call last):
  File "/home/klippy/moonraker/moonraker/server.py", line 562, in launch_server
    server = Server(app_args, log_manager, eventloop)
  File "/home/klippy/moonraker/moonraker/server.py", line 86, in __init__
    self.config = config = self._parse_config()
  File "/home/klippy/moonraker/moonraker/server.py", line 164, in _parse_config
    config = confighelper.get_configuration(self, self.app_args)
  File "/home/klippy/moonraker/moonraker/confighelper.py", line 1118, in get_configuration
    raise ConfigError("No section [server] in config")
moonraker.confighelper.ConfigError: No section [server] in config
2025-04-20 10:55:07,508 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 10:55:09 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 10:55:09,629 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 183 B
2025-04-20 10:55:09,629 [server.py:launch_server()] - Server Config Error
Traceback (most recent call last):
  File "/home/klippy/moonraker/moonraker/server.py", line 562, in launch_server
    server = Server(app_args, log_manager, eventloop)
  File "/home/klippy/moonraker/moonraker/server.py", line 86, in __init__
    self.config = config = self._parse_config()
  File "/home/klippy/moonraker/moonraker/server.py", line 164, in _parse_config
    config = confighelper.get_configuration(self, self.app_args)
  File "/home/klippy/moonraker/moonraker/confighelper.py", line 1118, in get_configuration
    raise ConfigError("No section [server] in config")
moonraker.confighelper.ConfigError: No section [server] in config
2025-04-20 10:55:09,629 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 10:55:12 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 10:55:12,753 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 183 B
2025-04-20 10:55:12,753 [server.py:launch_server()] - Server Config Error
Traceback (most recent call last):
  File "/home/klippy/moonraker/moonraker/server.py", line 562, in launch_server
    server = Server(app_args, log_manager, eventloop)
  File "/home/klippy/moonraker/moonraker/server.py", line 86, in __init__
    self.config = config = self._parse_config()
  File "/home/klippy/moonraker/moonraker/server.py", line 164, in _parse_config
    config = confighelper.get_configuration(self, self.app_args)
  File "/home/klippy/moonraker/moonraker/confighelper.py", line 1118, in get_configuration
    raise ConfigError("No section [server] in config")
moonraker.confighelper.ConfigError: No section [server] in config
2025-04-20 10:55:12,754 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:03:19 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:03:19,148 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 290 B
2025-04-20 11:03:19,148 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds
config_file = /home/klippy/printer_data/config/printer.cfg

[authorization]
trusted_clients = 
	192.168.178.0/24
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:03:19,167 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:03:19,168 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:03:19,170 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:03:19,313 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:03:19,319 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:03:19,333 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:03:19,334 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:03:19,334 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:03:19,354 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:03:19,391 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:03:19,394 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:03:19,402 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:03:19,403 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:03:19,403 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:03:19,405 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:03:19,429 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:03:19,429 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:03:19,432 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:03:19,432 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:03:19,433 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:03:19,433 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:03:19,434 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:03:19,437 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:03:19,437 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:03:19,438 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:03:19,440 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:03:19,444 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:03:19,444 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:03:19,444 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:03:19,448 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:03:19,451 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:03:19,454 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:03:19,456 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:03:19,456 [server.py:add_warning()] - Unparsed config option 'config_file: /home/klippy/printer_data/config/printer.cfg' detected in section [server].  This may be an option no longer available or could be the result of a module that failed to load.  In the future this will result in a startup error.
2025-04-20 11:03:19,456 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:03:19,460 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:03:19,460 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:03:19,460 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:03:19,460 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:03:19,461 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:03:19,461 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:03:19,463 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:03:19,463 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:03:19,463 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:03:19,463 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:03:19,464 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:03:19,465 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:03:19,465 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:03:19,466 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:03:19,466 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:03:19,467 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: d2ad794af199
2025-04-20 11:03:19,467 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:03:21,975 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:03:22,228 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:03:24,676 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:03:24,677 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 1.13ms
2025-04-20 11:03:24,677 [websockets.py:open()] - Websocket Opened: ID: 134101697705872, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:03:24,691 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:03:26,198 [websockets.py:on_close()] - Websocket Closed: ID: 134101697705872 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 1.52
2025-04-20 11:03:26,283 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:03:26,284 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.91ms
2025-04-20 11:03:26,284 [websockets.py:open()] - Websocket Opened: ID: 134101697707360, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:03:26,292 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:03:36,944 [websockets.py:on_close()] - Websocket Closed: ID: 134101697707360 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.66
2025-04-20 11:04:52,104 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:04:52,105 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:04:52,105 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:05:09,750 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.178.26
2025-04-20 11:05:09,898 [application.py:log_request()] - 404 GET /favicon.ico (192.168.178.26) [_TRUSTED_USER_] 2.50ms
2025-04-20 11:05:56,953 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:05:57,054 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:05:57,073 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:06:00 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:06:00,353 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 307 B
2025-04-20 11:06:00,353 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds
config_file = /home/klippy/printer_data/config/printer.cfg

[authorization]
enable = flase
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:06:00,371 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:06:00,372 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:06:00,374 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:06:00,517 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:06:00,523 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:06:00,537 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:06:00,537 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:06:00,538 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:06:00,557 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:06:00,593 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:06:00,596 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:06:00,604 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:06:00,604 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:

CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:06:00,605 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:06:00,607 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:06:00,632 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:06:00,632 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:06:00,635 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:06:00,635 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:06:00,636 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:06:00,636 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:06:00,637 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:06:00,640 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:06:00,640 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:06:00,641 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:06:00,643 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:06:00,647 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:06:00,647 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:06:00,647 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:06:00,651 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:06:00,654 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:06:00,657 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:06:00,659 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:06:00,659 [server.py:add_warning()] - Unparsed config option 'config_file: /home/klippy/printer_data/config/printer.cfg' detected in section [server].  This may be an option no longer available or could be the result of a module that failed to load.  In the future this will result in a startup error.
2025-04-20 11:06:00,659 [server.py:add_warning()] - Unparsed config option 'enable: flase' detected in section [authorization].  This may be an option no longer available or could be the result of a module that failed to load.  In the future this will result in a startup error.
2025-04-20 11:06:00,659 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:06:00,663 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:06:00,663 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:06:00,663 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:06:00,663 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:06:00,664 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:06:00,664 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:06:00,667 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:06:00,667 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:06:00,667 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:06:00,667 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:06:00,668 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:06:00,668 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:06:00,668 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:06:00,669 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:06:00,670 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:06:00,671 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: e42faf090a59
2025-04-20 11:06:00,671 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:06:02,631 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:06:02,632 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.97ms
2025-04-20 11:06:02,632 [websockets.py:open()] - Websocket Opened: ID: 130217234262016, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:06:02,642 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:06:03,180 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:06:03,432 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:06:04,034 [websockets.py:on_close()] - Websocket Closed: ID: 130217234262016 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 1.40
2025-04-20 11:06:04,124 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:06:04,125 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.97ms
2025-04-20 11:06:04,125 [websockets.py:open()] - Websocket Opened: ID: 130217234262064, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:06:04,134 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:06:04,934 [websockets.py:on_close()] - Websocket Closed: ID: 130217234262064 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 0.81
2025-04-20 11:06:05,005 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:06:05,006 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 1.07ms
2025-04-20 11:06:05,006 [websockets.py:open()] - Websocket Opened: ID: 130217234266048, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:06:05,015 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:06:15,933 [websockets.py:on_close()] - Websocket Closed: ID: 130217234266048 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.92
2025-04-20 11:07:33,329 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:07:33,329 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:07:33,329 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:08:41,406 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:08:41,507 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:08:41,512 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:08:43 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:08:43,409 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 291 B
2025-04-20 11:08:43,409 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
enabled = true
trusted_clients = 
	192.168.178.26
	192.168.178.28
	127.0.0.1
	localhost
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:08:43,427 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:08:43,428 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:08:43,430 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:08:43,574 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:08:43,580 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:08:43,594 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:08:43,594 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:08:43,595 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:08:43,614 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:08:43,651 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:08:43,654 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:08:43,662 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:08:43,662 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.26
192.168.178.28
127.0.0.1
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:08:43,663 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:08:43,665 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:08:43,690 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:08:43,690 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:08:43,692 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:08:43,692 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:08:43,694 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:08:43,694 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:08:43,695 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:08:43,698 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:08:43,698 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:08:43,699 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:08:43,701 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:08:43,704 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:08:43,705 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:08:43,705 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:08:43,709 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:08:43,712 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:08:43,715 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:08:43,717 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:08:43,717 [server.py:add_warning()] - Unparsed config option 'enabled: true' detected in section [authorization].  This may be an option no longer available or could be the result of a module that failed to load.  In the future this will result in a startup error.
2025-04-20 11:08:43,717 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:08:43,721 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:08:43,721 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:08:43,721 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:08:43,722 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:08:43,722 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:08:43,722 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:08:43,725 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:08:43,725 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:08:43,725 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:08:43,725 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:08:43,726 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:08:43,726 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:08:43,726 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:08:43,727 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:08:43,728 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:08:43,728 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: 2fed7d1899d8
2025-04-20 11:08:43,728 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:08:46,235 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:08:46,487 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:08:50,906 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:08:50,906 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 2.15ms
2025-04-20 11:08:50,907 [websockets.py:open()] - Websocket Opened: ID: 124197762400064, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:08:50,914 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:08:52,042 [websockets.py:on_close()] - Websocket Closed: ID: 124197762400064 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 1.14
2025-04-20 11:08:52,115 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:08:52,115 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.97ms
2025-04-20 11:08:52,115 [websockets.py:open()] - Websocket Opened: ID: 124197762399632, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:08:52,124 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:09:01,666 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:09:01,667 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 1.05ms
2025-04-20 11:09:01,667 [websockets.py:open()] - Websocket Opened: ID: 124197762396704, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36, Host Name: kobra.lan
2025-04-20 11:09:01,856 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:09:02,133 [websockets.py:on_close()] - Websocket Closed: ID: 124197762399632 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.02
2025-04-20 11:09:04,439 [websockets.py:on_close()] - Websocket Closed: ID: 124197762396704 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 2.77
2025-04-20 11:09:07,798 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.178.26
2025-04-20 11:10:16,386 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:10:16,387 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:10:16,388 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:11:50,492 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:11:50,594 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:11:50,612 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:11:52 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:11:52,514 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 276 B
2025-04-20 11:11:52,514 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	192.168.178.26
	192.168.178.28
	127.0.0.1
	localhost
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:11:52,532 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:11:52,534 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:11:52,535 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:11:52,676 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:11:52,682 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:11:52,696 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:11:52,697 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:11:52,697 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:11:52,716 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:11:52,753 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:11:52,756 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:11:52,764 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:11:52,764 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.26
192.168.178.28
127.0.0.1
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:11:52,765 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:11:52,767 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:11:52,790 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:11:52,790 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:11:52,793 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:11:52,793 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:11:52,794 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:11:52,794 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:11:52,796 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:11:52,798 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:11:52,798 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:11:52,799 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:11:52,802 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:11:52,805 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:11:52,805 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:11:52,806 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:11:52,809 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:11:52,813 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:11:52,816 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:11:52,818 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:11:52,818 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:11:52,822 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:11:52,822 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:11:52,822 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:11:52,822 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:11:52,823 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:11:52,823 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:11:52,826 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:11:52,826 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:11:52,826 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:11:52,826 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:11:52,827 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:11:52,828 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:11:52,828 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:11:52,829 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:11:52,830 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:11:52,830 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: 002895018ee3
2025-04-20 11:11:52,830 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:11:55,338 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:11:55,589 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:13:24,692 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:13:24,794 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:13:24,799 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:13:28 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:13:28,140 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 301 B
2025-04-20 11:13:28,140 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	192.168.178.26
	192.168.178.28
	127.0.0.1
	localhost
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

[cors]
allow_origin = *

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:13:28,159 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:13:28,161 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:13:28,162 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:13:28,308 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:13:28,314 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:13:28,328 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:13:28,328 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:13:28,329 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:13:28,348 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:13:28,386 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:13:28,390 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:13:28,398 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:13:28,398 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.26
192.168.178.28
127.0.0.1
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:13:28,399 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:13:28,401 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:13:28,425 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:13:28,425 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:13:28,427 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:13:28,428 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:13:28,429 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:13:28,429 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:13:28,430 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:13:28,433 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:13:28,433 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:13:28,434 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:13:28,436 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:13:28,440 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:13:28,440 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:13:28,440 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:13:28,444 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:13:28,447 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:13:28,450 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:13:28,452 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:13:28,452 [server.py:load_component()] - Unable to load component: (cors)
Traceback (most recent call last):
  File "/home/klippy/moonraker/moonraker/server.py", line 307, in load_component
    module = importlib.import_module(full_name)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'moonraker.components.cors'
2025-04-20 11:13:28,452 [server.py:add_warning()] - Unparsed config section [cors] detected.  This may be the result of a component that failed to load.  In the future this will result in a startup error.
2025-04-20 11:13:28,452 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:13:28,456 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:13:28,456 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:13:28,456 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:13:28,456 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:13:28,457 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:13:28,457 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:13:28,459 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:13:28,460 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:13:28,460 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:13:28,460 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:13:28,462 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:13:28,463 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:13:28,463 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:13:28,464 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:13:28,464 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:13:28,465 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: ab1fb92d353f
2025-04-20 11:13:28,465 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:13:30,973 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:13:31,225 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:13:32,059 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:13:32,059 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 1.88ms
2025-04-20 11:13:32,060 [websockets.py:open()] - Websocket Opened: ID: 138088600190832, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:13:32,068 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:13:33,225 [websockets.py:on_close()] - Websocket Closed: ID: 138088600190832 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 1.17
2025-04-20 11:13:33,295 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:13:33,295 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.91ms
2025-04-20 11:13:33,295 [websockets.py:open()] - Websocket Opened: ID: 138088600179984, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:13:33,303 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:13:34,850 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.178.26
2025-04-20 11:13:43,946 [websockets.py:on_close()] - Websocket Closed: ID: 138088600179984 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.65
2025-04-20 11:15:01,128 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:15:01,128 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:15:01,129 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:17:39,309 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:17:39,410 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:17:39,428 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:17:41 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:17:41,336 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 275 B
2025-04-20 11:17:41,336 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	192.168.178.26
	192.168.178.28
	127.0.0.1
	localhost
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:17:41,355 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:17:41,356 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:17:41,358 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:17:41,500 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:17:41,507 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:17:41,520 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:17:41,521 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:17:41,521 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:17:41,540 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:17:41,577 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:17:41,580 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:17:41,588 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:17:41,588 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.26
192.168.178.28
127.0.0.1
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:17:41,589 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:17:41,591 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:17:41,615 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:17:41,615 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:17:41,618 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:17:41,618 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:17:41,619 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:17:41,619 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:17:41,621 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:17:41,623 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:17:41,623 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:17:41,624 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:17:41,627 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:17:41,630 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:17:41,630 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:17:41,630 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:17:41,634 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:17:41,637 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:17:41,641 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:17:41,642 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:17:41,643 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:17:41,646 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:17:41,646 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:17:41,646 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:17:41,646 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:17:41,647 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:17:41,647 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:17:41,649 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:17:41,649 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:17:41,649 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:17:41,649 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:17:41,650 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:17:41,651 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:17:41,651 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:17:41,652 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:17:41,652 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:17:41,653 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: 5be7bbfc1049
2025-04-20 11:17:41,653 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:17:44,161 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:17:44,412 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:19:14,271 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:19:14,271 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:19:14,272 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:20:05,329 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:20:05,430 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:20:05,449 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:20:07 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:20:07,345 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 305 B
2025-04-20 11:20:07,345 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	192.168.178.26
	192.168.178.28
	127.0.0.1
	localhost
	172.18.0.1
	172.18.0.4
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:20:07,364 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:20:07,365 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:20:07,367 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:20:07,510 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:20:07,516 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:20:07,530 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:20:07,531 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:20:07,531 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:20:07,551 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:20:07,588 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:20:07,591 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:20:07,599 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:20:07,599 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.26
192.168.178.28
127.0.0.1
172.18.0.1
172.18.0.4
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:20:07,600 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:20:07,602 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:20:07,626 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:20:07,626 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:20:07,629 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:20:07,629 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:20:07,630 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:20:07,630 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:20:07,631 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:20:07,634 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:20:07,634 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:20:07,635 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:20:07,637 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:20:07,640 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:20:07,641 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:20:07,641 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:20:07,645 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:20:07,648 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:20:07,651 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:20:07,653 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:20:07,653 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:20:07,656 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:20:07,656 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:20:07,656 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:20:07,657 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:20:07,657 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:20:07,658 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:20:07,660 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.2
2025-04-20 11:20:07,660 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:20:07,660 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:20:07,660 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:20:07,661 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:20:07,662 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:20:07,662 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:20:07,663 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:20:07,663 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:20:07,664 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: fd7194a73a42
2025-04-20 11:20:07,664 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:20:10,172 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:20:10,424 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:20:17,095 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:20:17,096 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 1.89ms
2025-04-20 11:20:17,096 [websockets.py:open()] - Websocket Opened: ID: 133512074720976, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:20:17,104 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:20:18,306 [websockets.py:on_close()] - Websocket Closed: ID: 133512074720976 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 1.21
2025-04-20 11:20:18,376 [websockets.py:prepare()] - Websocket Failed Authentication: HTTP 401: Unauthorized (Unauthorized)
2025-04-20 11:20:18,376 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [No User] 0.97ms
2025-04-20 11:20:18,376 [websockets.py:open()] - Websocket Opened: ID: 133512074722752, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:20:18,386 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: server.connection.identify, Code: -32602, Message: Unauthorized
2025-04-20 11:20:21,438 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.178.26
2025-04-20 11:20:28,947 [websockets.py:on_close()] - Websocket Closed: ID: 133512074722752 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.57
2025-04-20 11:21:40,289 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:21:40,289 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:21:40,290 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:23:41,426 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:23:41,528 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:23:41,546 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:23:43 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:23:43,457 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 11:23:43,457 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:23:43,476 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:23:43,477 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:23:43,479 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:23:43,625 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:23:43,631 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:23:43,645 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:23:43,646 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:23:43,646 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:23:43,667 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:23:43,704 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:23:43,708 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:23:43,717 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:23:43,717 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:23:43,718 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:23:43,720 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:23:43,744 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:23:43,744 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:23:43,747 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:23:43,747 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:23:43,748 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:23:43,748 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:23:43,750 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:23:43,752 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:23:43,752 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:23:43,753 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:23:43,755 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:23:43,759 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:23:43,759 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:23:43,759 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:23:43,763 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:23:43,767 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:23:43,770 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:23:43,772 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:23:43,772 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:23:43,775 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:23:43,775 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:23:43,776 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:23:43,776 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:23:43,776 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:23:43,776 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:23:43,780 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.2
2025-04-20 11:23:43,780 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:23:43,780 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:23:43,780 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:23:43,781 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:23:43,782 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:23:43,782 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:23:43,783 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:23:43,783 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:23:43,784 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: aa00a69607b9
2025-04-20 11:23:43,784 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:23:45,532 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.178.26
2025-04-20 11:23:46,293 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:23:46,545 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:23:56,470 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 172.18.0.3
2025-04-20 11:23:56,471 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [_TRUSTED_USER_] 1.05ms
2025-04-20 11:23:56,471 [websockets.py:open()] - Websocket Opened: ID: 139773632958608, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:23:56,481 [websockets.py:_handle_identify()] - Websocket 139773632958608 Client Identified - Name: mainsail, Version: 2.13.2, Type: web
2025-04-20 11:23:56,536 [application.py:log_request()] - 404 GET /server/files/config/.theme/default.json?time=1745148235453 (172.18.0.3) [_TRUSTED_USER_] 1.55ms
2025-04-20 11:23:56,536 [application.py:log_request()] - 404 GET /server/files/config/.theme/maintenance.json?time=1745148235454 (172.18.0.3) [_TRUSTED_USER_] 1.77ms
2025-04-20 11:24:04,870 [websockets.py:on_close()] - Websocket Closed: ID: 139773632958608 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 8.40
2025-04-20 11:24:04,944 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [_TRUSTED_USER_] 0.85ms
2025-04-20 11:24:04,944 [websockets.py:open()] - Websocket Opened: ID: 139773632965568, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:24:04,951 [websockets.py:_handle_identify()] - Websocket 139773632965568 Client Identified - Name: mainsail, Version: 2.13.2, Type: web
2025-04-20 11:24:07,834 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:24:09,087 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:25:38,982 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:25:38,982 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:25:38,983 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:33:25,508 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:33:25,609 [websockets.py:on_close()] - Websocket Closed: ID: 139773632965568 Close Code: 1001, Close Reason: Server Shutdown, Pong Time Elapsed: 0.66
2025-04-20 11:33:25,610 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:33:25,628 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:33:27 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:33:27,273 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 11:33:27,273 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:33:27,285 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:33:27,286 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:33:27,288 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:33:27,422 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:33:27,426 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:33:27,429 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:33:27,430 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:33:27,430 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:33:27,451 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:33:27,458 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:33:27,462 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:33:27,466 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:33:27,466 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:33:27,467 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:33:27,468 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:33:27,477 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:33:27,477 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:33:27,477 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:33:27,477 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:33:27,479 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:33:27,479 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:33:27,479 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:33:27,480 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:33:27,480 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:33:27,480 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:33:27,481 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:33:27,481 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:33:27,481 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:33:27,482 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:33:27,483 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:33:27,484 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:33:27,485 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:33:27,486 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:33:27,486 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:33:27,502 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:33:27,502 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:33:27,502 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:33:27,503 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:33:27,503 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:33:27,503 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:33:27,506 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.2
2025-04-20 11:33:27,506 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:33:27,506 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:33:27,506 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:33:27,507 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:33:27,508 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:33:27,508 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:33:27,508 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:33:27,509 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:33:27,509 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: aa00a69607b9
2025-04-20 11:33:27,510 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:33:27,762 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:33:28,013 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:34:57,629 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:34:57,629 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:34:57,630 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:40:40,007 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:40:40,109 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:40:40,128 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:40:42 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:40:42,014 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 11:40:42,014 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:40:42,033 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:40:42,034 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:40:42,036 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:40:42,179 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:40:42,185 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:40:42,199 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:40:42,199 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:40:42,200 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:40:42,219 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:40:42,257 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:40:42,260 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:40:42,268 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:40:42,268 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:40:42,269 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:40:42,271 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:40:42,296 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:40:42,296 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:40:42,298 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:40:42,299 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:40:42,300 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:40:42,300 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:40:42,302 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:40:42,304 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:40:42,304 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:40:42,305 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:40:42,307 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:40:42,311 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:40:42,311 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:40:42,311 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:40:42,315 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:40:42,319 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:40:42,322 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:40:42,324 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:40:42,324 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:40:42,327 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:40:42,327 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:40:42,327 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:40:42,328 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:40:42,328 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:40:42,328 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:40:42,331 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.2
2025-04-20 11:40:42,331 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:40:42,331 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:40:42,331 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:40:42,332 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:40:42,333 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:40:42,333 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:40:42,334 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:40:42,335 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:40:42,335 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: 28f67aff3b82
2025-04-20 11:40:42,335 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:40:44,844 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:40:45,095 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:41:44,165 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:41:44,266 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:41:44,285 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:41:46 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:41:46,182 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 11:41:46,182 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:41:46,201 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:41:46,202 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:41:46,204 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:41:46,346 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:41:46,352 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:41:46,366 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:41:46,366 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:41:46,367 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:41:46,387 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:41:46,423 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:41:46,426 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:41:46,434 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:41:46,435 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:41:46,436 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:41:46,438 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:41:46,461 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:41:46,461 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:41:46,464 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:41:46,464 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:41:46,465 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:41:46,465 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:41:46,467 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:41:46,469 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:41:46,469 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:41:46,470 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:41:46,473 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:41:46,476 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:41:46,476 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:41:46,477 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:41:46,480 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:41:46,484 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:41:46,487 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:41:46,489 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:41:46,489 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:41:46,492 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:41:46,492 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:41:46,492 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:41:46,492 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:41:46,493 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:41:46,493 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:41:46,495 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.2
2025-04-20 11:41:46,496 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:41:46,496 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:41:46,496 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:41:46,497 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:41:46,498 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:41:46,498 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.2
2025-04-20 11:41:46,499 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:41:46,500 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:41:46,500 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: 8dc7185abd9a
2025-04-20 11:41:46,500 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:41:49,008 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:41:49,260 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:43:19,130 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:43:19,130 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:43:19,131 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:49:21,546 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:49:21,646 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:49:21,652 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:49:23 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:49:23,541 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 11:49:23,541 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:49:23,559 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:49:23,561 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:49:23,562 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:49:23,702 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:49:23,708 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:49:23,722 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:49:23,722 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:49:23,723 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:49:23,742 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:49:23,778 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:49:23,781 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:49:23,789 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:49:23,790 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:49:23,791 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:49:23,793 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:49:23,816 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:49:23,816 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:49:23,819 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:49:23,819 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:49:23,820 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:49:23,820 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:49:23,822 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:49:23,824 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:49:23,824 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:49:23,825 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:49:23,827 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:49:23,831 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:49:23,831 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:49:23,831 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:49:23,835 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:49:23,838 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:49:23,841 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:49:23,843 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:49:23,843 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:49:23,847 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:49:23,847 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:49:23,847 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:49:23,847 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:49:23,847 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:49:23,848 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:49:23,850 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:49:23,850 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:49:23,850 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:49:23,850 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:49:23,852 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:49:23,854 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:49:23,854 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:49:23,855 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:49:23,855 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:49:23,855 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: f9ba447ac04e
2025-04-20 11:49:23,855 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:49:26,363 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:49:26,615 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:49:35,665 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 172.18.0.3
2025-04-20 11:49:35,665 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [_TRUSTED_USER_] 1.34ms
2025-04-20 11:49:35,665 [websockets.py:open()] - Websocket Opened: ID: 135853905857360, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:49:35,677 [websockets.py:_handle_identify()] - Websocket 135853905857360 Client Identified - Name: mainsail, Version: 2.13.2, Type: web
2025-04-20 11:49:39,873 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:49:41,127 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:49:44,137 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.178.26
2025-04-20 11:49:58,466 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:49:59,720 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:51:29,599 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:51:29,599 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:51:29,600 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:52:03,639 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:52:03,741 [websockets.py:on_close()] - Websocket Closed: ID: 135853905857360 Close Code: 1001, Close Reason: Server Shutdown, Pong Time Elapsed: 8.07
2025-04-20 11:52:03,742 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:52:03,759 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:52:05 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:52:05,426 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 11:52:05,426 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:52:05,438 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:52:05,439 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:52:05,441 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:52:05,575 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:52:05,579 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:52:05,583 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:52:05,583 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:52:05,584 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:52:05,603 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:52:05,610 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:52:05,614 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:52:05,618 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:52:05,618 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:52:05,619 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:52:05,620 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:52:05,629 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:52:05,629 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:52:05,629 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:52:05,629 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:52:05,631 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:52:05,631 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:52:05,631 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:52:05,632 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:52:05,632 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:52:05,632 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:52:05,633 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:52:05,633 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:52:05,633 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:52:05,634 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:52:05,635 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:52:05,636 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:52:05,637 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:52:05,638 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:52:05,638 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:52:05,654 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:52:05,654 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:52:05,654 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:52:05,655 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:52:05,655 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:52:05,655 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:52:05,658 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:52:05,658 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:52:05,658 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:52:05,658 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:52:05,659 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:52:05,660 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:52:05,660 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:52:05,661 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:52:05,661 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:52:05,662 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: f9ba447ac04e
2025-04-20 11:52:05,662 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:52:05,916 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:52:06,168 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:52:36,641 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 172.18.0.3
2025-04-20 11:52:36,642 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [_TRUSTED_USER_] 1.21ms
2025-04-20 11:52:36,642 [websockets.py:open()] - Websocket Opened: ID: 138724250796240, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:52:36,652 [websockets.py:_handle_identify()] - Websocket 138724250796240 Client Identified - Name: mainsail, Version: 2.13.2, Type: web
2025-04-20 11:52:42,194 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:52:43,450 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:53:05,701 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.178.26
2025-04-20 11:54:13,334 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:54:13,334 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:54:13,335 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:54:57,384 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:54:57,486 [websockets.py:on_close()] - Websocket Closed: ID: 138724250796240 Close Code: 1001, Close Reason: Server Shutdown, Pong Time Elapsed: 0.84
2025-04-20 11:54:57,486 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:54:57,504 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:54:59 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:54:59,386 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 11:54:59,386 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:54:59,404 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:54:59,405 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:54:59,407 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:54:59,545 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:54:59,552 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:54:59,565 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:54:59,565 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:54:59,566 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:54:59,585 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:54:59,622 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:54:59,625 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:54:59,633 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:54:59,633 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:54:59,634 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:54:59,636 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:54:59,659 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:54:59,659 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:54:59,661 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:54:59,661 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:54:59,663 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:54:59,663 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:54:59,664 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:54:59,666 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:54:59,667 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:54:59,668 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:54:59,671 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:54:59,674 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:54:59,674 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:54:59,675 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:54:59,678 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:54:59,682 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:54:59,685 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:54:59,687 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:54:59,687 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:54:59,690 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:54:59,690 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:54:59,690 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:54:59,691 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:54:59,691 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:54:59,691 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:54:59,693 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:54:59,694 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:54:59,694 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:54:59,694 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:54:59,695 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:54:59,695 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:54:59,695 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:54:59,696 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:54:59,697 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:54:59,697 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: 6d1d844680e5
2025-04-20 11:54:59,697 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:55:02,205 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:55:02,457 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:56:32,324 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 11:56:32,325 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 11:56:32,325 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 11:58:51,489 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 11:58:51,590 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 11:58:51,608 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 11:58:53 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 11:58:53,489 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 11:58:53,489 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 11:58:53,507 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 11:58:53,509 [server.py:load_component()] - Component (template) loaded
2025-04-20 11:58:53,510 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 11:58:53,652 [server.py:load_component()] - Component (application) loaded
2025-04-20 11:58:53,658 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 11:58:53,672 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 11:58:53,672 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 11:58:53,673 [server.py:load_component()] - Component (database) loaded
2025-04-20 11:58:53,692 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 11:58:53,729 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 11:58:53,732 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 11:58:53,739 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 11:58:53,740 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 11:58:53,741 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 11:58:53,743 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 11:58:53,766 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 11:58:53,766 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 11:58:53,769 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 11:58:53,769 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 11:58:53,770 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 11:58:53,770 [server.py:load_component()] - Component (machine) loaded
2025-04-20 11:58:53,772 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 11:58:53,774 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 11:58:53,774 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 11:58:53,775 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 11:58:53,777 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 11:58:53,781 [database.py:register_table()] - Found registered table job_history
2025-04-20 11:58:53,781 [database.py:register_table()] - Found registered table job_totals
2025-04-20 11:58:53,781 [server.py:load_component()] - Component (history) loaded
2025-04-20 11:58:53,785 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 11:58:53,788 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 11:58:53,791 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 11:58:53,793 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 11:58:53,793 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 11:58:53,796 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 11:58:53,797 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 11:58:53,797 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 11:58:53,797 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 11:58:53,797 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 11:58:53,797 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 11:58:53,800 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 11:58:53,800 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 11:58:53,800 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 11:58:53,800 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:58:53,801 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 11:58:53,802 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 11:58:53,802 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 11:58:53,803 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 11:58:53,803 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 11:58:53,804 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: acfb785abff9
2025-04-20 11:58:53,804 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 11:58:56,312 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 11:58:56,564 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 11:59:08,912 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 172.18.0.3
2025-04-20 11:59:08,912 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [_TRUSTED_USER_] 1.17ms
2025-04-20 11:59:08,912 [websockets.py:open()] - Websocket Opened: ID: 137684424445952, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 11:59:08,926 [websockets.py:_handle_identify()] - Websocket 137684424445952 Client Identified - Name: mainsail, Version: 2.13.2, Type: web
2025-04-20 12:00:26,463 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 12:00:26,463 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 12:00:26,464 [klippy_connection.py:_check_ready()] - 
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 12:03:53,703 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 12:03:53,805 [websockets.py:on_close()] - Websocket Closed: ID: 137684424445952 Close Code: 1001, Close Reason: Server Shutdown, Pong Time Elapsed: 4.89
2025-04-20 12:03:53,806 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 12:03:53,823 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 12:03:55 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 12:03:55,703 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 12:03:55,704 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 12:03:55,722 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 12:03:55,723 [server.py:load_component()] - Component (template) loaded
2025-04-20 12:03:55,724 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 12:03:55,867 [server.py:load_component()] - Component (application) loaded
2025-04-20 12:03:55,873 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 12:03:55,887 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 12:03:55,887 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 12:03:55,887 [server.py:load_component()] - Component (database) loaded
2025-04-20 12:03:55,907 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 12:03:55,951 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 12:03:55,954 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 12:03:55,962 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 12:03:55,963 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 12:03:55,963 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 12:03:55,966 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 12:03:55,991 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 12:03:55,991 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 12:03:55,993 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 12:03:55,993 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 12:03:55,995 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 12:03:55,995 [server.py:load_component()] - Component (machine) loaded
2025-04-20 12:03:55,996 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 12:03:55,999 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 12:03:55,999 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 12:03:56,000 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 12:03:56,002 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 12:03:56,006 [database.py:register_table()] - Found registered table job_history
2025-04-20 12:03:56,006 [database.py:register_table()] - Found registered table job_totals
2025-04-20 12:03:56,006 [server.py:load_component()] - Component (history) loaded
2025-04-20 12:03:56,010 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 12:03:56,013 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 12:03:56,016 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 12:03:56,018 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 12:03:56,018 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 12:03:56,022 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 12:03:56,022 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 12:03:56,022 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 12:03:56,023 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 12:03:56,023 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 12:03:56,023 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 12:03:56,026 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 12:03:56,026 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 12:03:56,026 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 12:03:56,026 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 12:03:56,027 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 12:03:56,028 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 12:03:56,028 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 12:03:56,029 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 12:03:56,029 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 12:03:56,030 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: 2d261ce07f7c
2025-04-20 12:03:56,030 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 12:03:58,539 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 12:03:58,791 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 12:04:00,052 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 12:04:00,052 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 12:04:00,303 [job_state.py:_handle_started()] - Job state initialized: standby
2025-04-20 12:04:00,304 [klippy_connection.py:_verify_klippy_requirements()] - 
Warning, unable to detect the following printer objects:
[display_status]
Please add the the above sections to printer.cfg for full Moonraker functionality.
2025-04-20 12:04:00,556 [klippy_connection.py:_check_ready()] - Klippy ready
2025-04-20 12:04:01,055 [data_store.py:_init_sensors()] - Configuring available sensors: ['heater_bed', 'extruder']
2025-04-20 12:04:34,432 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 172.18.0.3
2025-04-20 12:04:34,432 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [_TRUSTED_USER_] 0.99ms
2025-04-20 12:04:34,432 [websockets.py:open()] - Websocket Opened: ID: 129025859401568, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 12:04:34,446 [websockets.py:_handle_identify()] - Websocket 129025859401568 Client Identified - Name: mainsail, Version: 2.13.2, Type: web
2025-04-20 12:07:42,670 [server.py:_handle_term_signal()] - Exiting with signal SIGTERM
2025-04-20 12:07:42,773 [websockets.py:on_close()] - Websocket Closed: ID: 129025859401568 Close Code: 1001, Close Reason: Server Shutdown, Pong Time Elapsed: 8.34
2025-04-20 12:07:42,774 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 12:07:42,791 [server.py:main()] - Server Shutdown
-------------------- Log Start | Sun Apr 20 12:07:44 2025 --------------------
platform: Linux-6.8.0-57-generic-x86_64-with-glibc2.36
data_path: /home/klippy/printer_data
is_default_data_path: False
config_file: /home/klippy/printer_data/config/moonraker.conf
backup_config: None
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: a56097d434a74aa8a1ffe99900bbd80c
unix_socket_path: /home/klippy/printer_data/comms/moonraker.sock
structured_logging: False
software_version: v0.9.3-73-g5ee0adc
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/klippy/printer_data/logs/moonraker.log
python_version: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]
launch_args: /home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/moonraker.py -d /home/klippy/printer_data
msgspec_enabled: False
uvloop_enabled: False
2025-04-20 12:07:44,679 [confighelper.py:read_file()] - Configuration File '/home/klippy/printer_data/config/moonraker.conf' parsed, total size: 262 B
2025-04-20 12:07:44,679 [server.py:add_log_rollover_item()] - 
#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /tmp/klippy_uds

[authorization]
trusted_clients = 
	localhost
	192.168.178.0/24
	172.18.0.0/16
cors_domains = 
	http://kobra.lan
	https://kobra.lan
	http://localhost
	http://127.0.0.1

#################################################################
All Configuration Files:
/home/klippy/printer_data/config/moonraker.conf
#################################################################
2025-04-20 12:07:44,697 [server.py:load_component()] - Component (secrets) loaded
2025-04-20 12:07:44,698 [server.py:load_component()] - Component (template) loaded
2025-04-20 12:07:44,700 [server.py:load_component()] - Component (klippy_connection) loaded
2025-04-20 12:07:44,839 [server.py:load_component()] - Component (application) loaded
2025-04-20 12:07:44,846 [server.py:load_component()] - Component (websockets) loaded
2025-04-20 12:07:44,859 [server.py:add_log_rollover_item()] - Loading Sqlite database provider. Sqlite Version: 3.40.1
2025-04-20 12:07:44,860 [server.py:add_log_rollover_item()] - Unsafe Shutdown Count: 6
2025-04-20 12:07:44,860 [server.py:load_component()] - Component (database) loaded
2025-04-20 12:07:44,879 [server.py:load_component()] - Component (dbus_manager) loaded
2025-04-20 12:07:44,916 [file_manager.py:__init__()] - Using File System Observer: inotify
2025-04-20 12:07:44,919 [server.py:load_component()] - Component (file_manager) loaded
2025-04-20 12:07:44,927 [database.py:register_table()] - Found registered table authorized_users
2025-04-20 12:07:44,928 [authorization.py:__init__()] - Authorization Configuration Loaded
Trusted Clients:
192.168.178.0/24
172.18.0.0/16
localhost
CORS Domains:
http://kobra\.lan
https://kobra\.lan
http://localhost
http://127\.0\.0\.1
2025-04-20 12:07:44,928 [server.py:load_component()] - Component (authorization) loaded
2025-04-20 12:07:44,931 [server.py:load_component()] - Component (klippy_apis) loaded
2025-04-20 12:07:44,954 [machine.py:_check_inside_container()] - Container detected via sched: docker
2025-04-20 12:07:44,954 [server.py:add_log_rollover_item()] - 
System Info:

***python***
  version: (3, 10, 17, 'final', 0)
  version_string: 3.10.17 (main, Apr  9 2025, 18:21:43) [GCC 12.2.0]

***cpu_info***
  cpu_count: 16
  bits: 64bit
  processor: x86_64
  cpu_desc: AMD Ryzen 7 5700U with Radeon Graphics
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 32245508
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.8.0-57-generic

***virtualization***
  virt_type: container
  virt_identifier: docker

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
2025-04-20 12:07:44,956 [server.py:load_component()] - Component (shell_command) loaded
2025-04-20 12:07:44,956 [machine.py:__init__()] - Using System Provider: systemd_dbus
2025-04-20 12:07:44,958 [server.py:add_log_rollover_item()] - Module libcamera unavailable, import failed
2025-04-20 12:07:44,958 [server.py:load_component()] - Component (machine) loaded
2025-04-20 12:07:44,959 [server.py:load_component()] - Component (data_store) loaded
2025-04-20 12:07:44,961 [proc_stats.py:__init__()] - Unable to find 'vcgencmd', throttle checking disabled
2025-04-20 12:07:44,962 [server.py:load_component()] - Component (proc_stats) loaded
2025-04-20 12:07:44,962 [server.py:load_component()] - Component (job_state) loaded
2025-04-20 12:07:44,965 [server.py:load_component()] - Component (job_queue) loaded
2025-04-20 12:07:44,968 [database.py:register_table()] - Found registered table job_history
2025-04-20 12:07:44,968 [database.py:register_table()] - Found registered table job_totals
2025-04-20 12:07:44,969 [server.py:load_component()] - Component (history) loaded
2025-04-20 12:07:44,972 [server.py:load_component()] - Component (http_client) loaded
2025-04-20 12:07:44,976 [server.py:load_component()] - Component (announcements) loaded
2025-04-20 12:07:44,979 [server.py:load_component()] - Component (webcam) loaded
2025-04-20 12:07:44,981 [server.py:load_component()] - Component (extensions) loaded
2025-04-20 12:07:44,981 [server.py:_initialize_component()] - Performing Component Post Init: [database]
2025-04-20 12:07:44,984 [server.py:_initialize_component()] - Performing Component Post Init: [dbus_manager]
2025-04-20 12:07:44,984 [dbus_manager.py:component_init()] - Unable to Connect to D-Bus
2025-04-20 12:07:44,985 [server.py:_initialize_component()] - Performing Component Post Init: [authorization]
2025-04-20 12:07:44,985 [server.py:_initialize_component()] - Performing Component Post Init: [machine]
2025-04-20 12:07:44,985 [machine.py:validation_init()] - Installation version in database up to date
2025-04-20 12:07:44,985 [server.py:add_warning()] - [machine]: DBus Connection Not available, systemd  service tracking and actions are disabled
2025-04-20 12:07:44,988 [machine.py:_find_public_ip()] - Detected Local IP: 172.18.0.4
2025-04-20 12:07:44,988 [server.py:_initialize_component()] - Performing Component Post Init: [proc_stats]
2025-04-20 12:07:44,988 [server.py:_initialize_component()] - Performing Component Post Init: [history]
2025-04-20 12:07:44,988 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 12:07:44,990 [server.py:_initialize_component()] - Performing Component Post Init: [announcements]
2025-04-20 12:07:44,992 [server.py:_initialize_component()] - Performing Component Post Init: [webcam]
2025-04-20 12:07:44,992 [webcam.py:_set_default_host_ip()] - Default public webcam address set: http://172.18.0.4
2025-04-20 12:07:44,993 [server.py:_initialize_component()] - Performing Component Post Init: [klippy_connection]
2025-04-20 12:07:44,993 [extensions.py:start_unix_server()] - Creating Unix Domain Socket at '/home/klippy/printer_data/comms/moonraker.sock'
2025-04-20 12:07:44,993 [server.py:start_server()] - Starting Moonraker on (0.0.0.0, 7125), Hostname: b6446d06d7af
2025-04-20 12:07:44,993 [application.py:listen()] - SSL Certificate/Key not configured, aborting HTTPS Server startup
2025-04-20 12:07:47,501 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 12:07:47,752 [server.py:add_log_rollover_item()] - Klipper Version: 0ba9ab7
2025-04-20 12:07:49,011 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 12:07:49,012 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 12:07:49,262 [job_state.py:_handle_started()] - Job state initialized: standby
2025-04-20 12:07:49,515 [klippy_connection.py:_check_ready()] - Klippy ready
2025-04-20 12:07:50,015 [data_store.py:_init_sensors()] - Configuring available sensors: ['heater_bed', 'extruder']
2025-04-20 12:07:55,808 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 172.18.0.3
2025-04-20 12:07:55,808 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [_TRUSTED_USER_] 1.03ms
2025-04-20 12:07:55,809 [websockets.py:open()] - Websocket Opened: ID: 131053919639232, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 12:07:55,811 [websockets.py:_handle_identify()] - Websocket 131053919639232 Client Identified - Name: mainsail, Version: 2.13.2, Type: web
2025-04-20 12:09:02,795 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.gcode.script, Code: 400, Message: Extrude below minimum temp
See the 'min_extrude_temp' config option for details
2025-04-20 12:09:38,391 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.gcode.script, Code: 400, Message: Extrude below minimum temp
See the 'min_extrude_temp' config option for details
2025-04-20 12:10:46,507 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.gcode.script, Code: 400, Message: Extrude below minimum temp
See the 'min_extrude_temp' config option for details
2025-04-20 12:20:22,224 [application.py:prepare()] - Upload Request Received from 172.18.0.3
Content-Type: multipart/form-data; boundary=----WebKitFormBoundary8QiSQzdLBWCjsHpv
2025-04-20 12:20:22,231 [application.py:post()] - Processing Uploaded File: CE3PRO_universal-door-slider.gcode
2025-04-20 12:20:22,303 [shell_command.py:pipe_data_received()] - INFO:metadata:Processor preprocess_cancellation is disabled

2025-04-20 12:20:22,316 [shell_command.py:_check_proc_success()] - Command (/home/klippy/klippy-env/bin/python /home/klippy/moonraker/moonraker/components/file_manager/metadata.py -c /tmp/metacfg-e1j1tbjx.json) successfully finished
2025-04-20 12:20:22,318 [application.py:log_request()] - 201 POST /server/files/upload (172.18.0.3) [_TRUSTED_USER_] 94.07ms
2025-04-20 12:20:27,631 [klippy_apis.py:start_print()] - Requesting Job Start, filename = CE3PRO_universal-door-slider.gcode
2025-04-20 12:20:27,775 [job_state.py:_status_update()] - Job Started: CE3PRO_universal-door-slider.gcode
2025-04-20 13:22:44,998 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 172.18.0.3
2025-04-20 13:49:29,282 [websockets.py:on_close()] - Websocket Closed: ID: 131053919639232 Close Code: None, Close Reason: None, Pong Time Elapsed: 3.47
2025-04-20 13:49:33,100 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 172.18.0.3
2025-04-20 13:49:33,100 [application.py:log_request()] - 101 GET /websocket (172.18.0.3) [_TRUSTED_USER_] 0.98ms
2025-04-20 13:49:33,100 [websockets.py:open()] - Websocket Opened: ID: 131053919645520, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 OPR/117.0.0.0, Host Name: kobra.lan
2025-04-20 13:49:33,102 [websockets.py:_handle_identify()] - Websocket 131053919645520 Client Identified - Name: mainsail, Version: 2.13.2, Type: web
2025-04-20 13:56:38,666 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.gcode.script, Code: 400, Message: Shutdown due to M112 command
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Printer is shutdown

2025-04-20 13:56:38,834 [klippy_connection.py:_process_status_update()] - Klippy has shutdown
2025-04-20 13:56:38,836 [proc_stats.py:_handle_shutdown()] - 
Moonraker System Usage Statistics:
System Time: 1745157369.669258, Usage: 0.66%, Memory: 51756 kB
System Time: 1745157370.670412, Usage: 0.62%, Memory: 51756 kB
System Time: 1745157371.671366, Usage: 0.76%, Memory: 51756 kB
System Time: 1745157372.672341, Usage: 0.66%, Memory: 51756 kB
System Time: 1745157373.673229, Usage: 0.75%, Memory: 51756 kB
System Time: 1745157374.674141, Usage: 0.68%, Memory: 51756 kB
System Time: 1745157375.674964, Usage: 0.65%, Memory: 51756 kB
System Time: 1745157376.676084, Usage: 0.65%, Memory: 51756 kB
System Time: 1745157377.676207, Usage: 0.67%, Memory: 51756 kB
System Time: 1745157378.676965, Usage: 0.62%, Memory: 51756 kB
System Time: 1745157379.677816, Usage: 0.69%, Memory: 51756 kB
System Time: 1745157380.678864, Usage: 0.65%, Memory: 51756 kB
System Time: 1745157381.679826, Usage: 0.79%, Memory: 51756 kB
System Time: 1745157382.680875, Usage: 0.67%, Memory: 51756 kB
System Time: 1745157383.682439, Usage: 0.74%, Memory: 51756 kB
System Time: 1745157384.684644, Usage: 0.66%, Memory: 51756 kB
System Time: 1745157385.685498, Usage: 0.65%, Memory: 51756 kB
System Time: 1745157386.685881, Usage: 0.66%, Memory: 51756 kB
System Time: 1745157387.686051, Usage: 0.65%, Memory: 51756 kB
System Time: 1745157388.686937, Usage: 0.65%, Memory: 51756 kB
System Time: 1745157389.687300, Usage: 0.64%, Memory: 51756 kB
System Time: 1745157390.687906, Usage: 0.64%, Memory: 51756 kB
System Time: 1745157391.688920, Usage: 0.75%, Memory: 51756 kB
System Time: 1745157392.689554, Usage: 0.67%, Memory: 51756 kB
System Time: 1745157393.690770, Usage: 0.73%, Memory: 51756 kB
System Time: 1745157394.691806, Usage: 0.65%, Memory: 51756 kB
System Time: 1745157395.692633, Usage: 0.64%, Memory: 51756 kB
System Time: 1745157396.693709, Usage: 0.65%, Memory: 51756 kB
System Time: 1745157397.694081, Usage: 0.72%, Memory: 51756 kB
System Time: 1745157398.694697, Usage: 0.86%, Memory: 51756 kB
CPU Temperature: None
2025-04-20 13:56:46,154 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 13:56:47,409 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 13:56:48,165 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 13:56:48,165 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 13:56:48,166 [klippy_connection.py:_check_ready()] - 
Can not update MCU 'mcu' config as it is shutdown
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

2025-04-20 13:56:51,541 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-04-20 13:56:52,794 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-04-20 13:56:54,052 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-04-20 13:56:54,053 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-04-20 13:56:54,303 [job_state.py:_handle_started()] - Job state initialized: standby
2025-04-20 13:56:54,556 [klippy_connection.py:_check_ready()] - Klippy ready
2025-04-20 13:56:55,059 [data_store.py:_init_sensors()] - Configuring available sensors: ['heater_bed', 'extruder']
2025-04-20 13:57:03,556 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.gcode.script, Code: 400, Message: Must home axis first: 0.000 0.000 10.000 [0.000]
2025-04-20 13:57:31,167 [klippy_connection.py:_process_status_update()] - Klippy has shutdown
2025-04-20 13:57:31,168 [proc_stats.py:_handle_shutdown()] - 
Moonraker System Usage Statistics:
System Time: 1745157421.712602, Usage: 0.77%, Memory: 51772 kB
System Time: 1745157422.712895, Usage: 0.68%, Memory: 51772 kB
System Time: 1745157423.713849, Usage: 0.93%, Memory: 51772 kB
System Time: 1745157424.714830, Usage: 0.66%, Memory: 51772 kB
System Time: 1745157425.715555, Usage: 0.64%, Memory: 51772 kB
System Time: 1745157426.716707, Usage: 0.63%, Memory: 51772 kB
System Time: 1745157427.717733, Usage: 0.64%, Memory: 51772 kB
System Time: 1745157428.719771, Usage: 0.65%, Memory: 51772 kB
System Time: 1745157429.720891, Usage: 0.66%, Memory: 51772 kB
System Time: 1745157430.721824, Usage: 0.64%, Memory: 51772 kB
System Time: 1745157431.722970, Usage: 0.77%, Memory: 51772 kB
System Time: 1745157432.723126, Usage: 0.64%, Memory: 51772 kB
System Time: 1745157433.723732, Usage: 0.7%, Memory: 51772 kB
System Time: 1745157434.724738, Usage: 0.63%, Memory: 51772 kB
System Time: 1745157435.726818, Usage: 0.61%, Memory: 51772 kB
System Time: 1745157436.727821, Usage: 0.61%, Memory: 51772 kB
System Time: 1745157437.728322, Usage: 0.61%, Memory: 51772 kB
System Time: 1745157438.729278, Usage: 0.62%, Memory: 51772 kB
System Time: 1745157439.730155, Usage: 0.65%, Memory: 51772 kB
System Time: 1745157440.730817, Usage: 0.65%, Memory: 51772 kB
System Time: 1745157441.731599, Usage: 0.74%, Memory: 51772 kB
System Time: 1745157442.733710, Usage: 0.68%, Memory: 51772 kB
System Time: 1745157443.734741, Usage: 0.69%, Memory: 51772 kB
System Time: 1745157444.735829, Usage: 0.62%, Memory: 51772 kB
System Time: 1745157445.736738, Usage: 0.6%, Memory: 51772 kB
System Time: 1745157446.737759, Usage: 0.67%, Memory: 51772 kB
System Time: 1745157447.738600, Usage: 0.6%, Memory: 51772 kB
System Time: 1745157448.739626, Usage: 0.61%, Memory: 51772 kB
System Time: 1745157449.740503, Usage: 0.59%, Memory: 51772 kB
System Time: 1745157450.742453, Usage: 0.61%, Memory: 51772 kB
CPU Temperature: None
2025-04-20 14:52:45,013 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 172.18.0.3
2025-04-20 15:00:24,353 [websockets.py:on_close()] - Websocket Closed: ID: 131053919645520 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 1.25
